manifests/plugin/remote_execution/ssh.pp

```diff
@@ -63,4 +63,17 @@
       creates => $ssh_identity_path,
     }
   }
+
+  case $::osfamily {
+    'Debian': {
+      $core_package = 'ruby-smart-proxy-remote-execution-ssh-core'
+    }
+    default: {
+      $core_package = 'tfm-rubygem-smart_proxy_remote_execution_ssh_core'
+    }
+  }
+
+  package { $core_package:
+    ensure => installed',
+  }
```

Comment:
It's the first time anything SCL is going on a smart proxy that I know of.

So it's not really even strictly a smart proxy plugin, but a component used by the smart proxy plugin so it can use the newer ruby. It's otherwise following the same naming convention, etc of the plugins, a so which is why I was wondering if it would be OK to use the plugin define as it handles all the various cases for the OS's.  It would just need to be extended to be scl aware.